images/base/files/usr/local/bin/entrypoint

```diff
@@ -88,6 +88,25 @@ fix_product_name() {
   fi
 }
 
+fix_product_uuid() {
+  # The system UUID is usually read from DMI via sysfs, the problem is that
+  # in the kind case this means that all (container) nodes share the same
+  # system/product uuid, as they share the same DMI.
+  # Note: The UUID is read from DMI, this tool is overwriting the sysfs files
+  # which should fix the attached issue, but this workaround does not address
+  # the issue if a tool is reading directly from DMI.
+  # https://github.com/kubernetes-sigs/kind/issues/1027
+  [[ ! -f /kind/product_uuid ]] && cat /proc/sys/kernel/random/uuid > /kind/product_uuid
```

Comment:
minor tweak: 
- create IFF the file doesn't already exist (for eventual restart node support in kind...)
- decouple the two IFs below by bind mounting both from this faked file